### PR TITLE
Dac6 15

### PR DESCRIPTION
**Pull request by daniel-manning**
*Submit file to backend for transformation
*store user's generated ids

### CODE REVIEW

File: app/models/ImportInstruction.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+case object New extends ImportInstruction
+case object Add extends ImportInstruction
+case object Replace extends ImportInstruction
+
+object ImportInstruction {
+  def apply(name: String): ImportInstruction = name match {
+    case "DAC6NEW" => New
+    case "DAC6ADD" => Add
+    case "DAC6REP" => Replace
+    //case _ => new IllegalArgumentException("Unrecognised Import Instruction")
```
Comment (helensoniega) [inline app/models/ImportInstruction.scala:30]: Missing 'DAC6DEL'

File: app/connectors/CrossBorderArrangementsConnector.scala [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  val submitUrl = s"${configuration.crossBorderArrangementsUrl}/disclose-cross-border-arrangements/submit"
+
+  private val headers = Seq(
+    HeaderNames.CONTENT_TYPE -> "application/xml"
+  )
+
+  def submitDocument(fileName: String, xmlDocument: Elem)(implicit hc: HeaderCarrier): Future[GeneratedIDs] =
+    httpClient.POST[String, GeneratedIDs](submitUrl, constructSubmission(fileName, xmlDocument).toString(), headers)
+
+/*  def createFormData( fileName: String,
```
Comment (ers2015112) [inline app/connectors/CrossBorderArrangementsConnector.scala:42]: Are these to do comments or redundant?

Comment (daniel-manning) [reply app/connectors/CrossBorderArrangementsConnector.scala]: removed, not needed in the end

File: app/models/ImportInstruction.scala (hunk 1 above)
Comment (daniel-manning) [reply app/models/ImportInstruction.scala]: added

**Pull request merged**